src/gui/redux/documentation/documentation.js

```diff
@@ -1,3 +1,3 @@
-import factory from 'trampss-redux-factory'
+import { simpleObject } from 'trampss-redux-factory'
 
-export default factory({ name: 'documentation', type: 'uniq' })
+export default simpleObject('documentation')
```

Comment:
I know but I wanted to upgrade the lib for my interest and the project didn't work. So I quicked fix to have a stable PR.